lt/client.go

```diff
@@ -793,23 +798,25 @@ func (u *user) Connect(stopCh chan struct{}, channelType model.ChannelType) erro
 	u.userID = user.Id
 
 	// join team
-	ctx, cancel = context.WithTimeout(context.Background(), httpRequestTimeout)
-	defer cancel()
-	_, _, err = client.AddTeamMember(ctx, u.cfg.teamID, user.Id)
-	if err != nil {
-		return err
-	}
-	cancel()
-
-	if channelType == "O" || channelType == "P" {
-		// join channel
+	if !u.cfg.noSetup {
```

Comment:
I find negative tests a little ugly, but I guess it makes sense here.

Reply:
We could in theory have a `-setup` flag default to `true` but not sure if it becomes any clearer.

Reply:
It's a double negative, too. Even worse...

Reply:
Sorry, I meant the current state is a double negative. `if u.cfg.setup {` is great.

Reply:
Cool, should be clearer now.